src/main/java/persistence/entity/SimpleEntityManger.java

```diff
@@ -1,30 +1,97 @@
 package persistence.entity;
 
+import jakarta.persistence.EntityExistsException;
+import persistence.sql.model.PKColumn;
+import persistence.sql.model.Table;
+
+import java.util.Objects;
+
 public class SimpleEntityManger implements EntityManager {
```

Comment:
```suggestion
public class SimpleEntityManager implements EntityManager {
```

Manager 를 Manger 로 쓰는 사소한 오타를 발견해서 공유합니다.

Reply:
감사합니다!